examples/esp32-wroom-32/test_ds18b20/main.c

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2022 Eduardo Azócar <eduardo@turpialdev.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @file    main.c
+ * @author  Eduardo Azócar <eduardo@turpialdev.com>
+ * @brief   Example_of_ds18
+ * @version 0.1
+ * @date    2022-01-04
+ *
+ * @copyright Copyright (c) 2022
```

Comment:
Same here

Reply:
sorry those last lines was generated by my vscode, i will fix it